internal/server/create_document.go

```diff
@@ -3,8 +3,10 @@ package server
 import (
 	"github.com/ministryofjustice/opg-go-common/template"
 	"github.com/ministryofjustice/opg-sirius-lpa-frontend/internal/sirius"
+	"golang.org/x/exp/slices"
```

Comment:
I thought the contains function below was handy and saved me writing a function to do it but don't know if requiring an extra package would have an affect on performance..